### ISSUE TITLE
overlay text - organizations logo and facbook and RSS logo

### ISSUE THREAD

**Issue by zomer-g** [OriHoch/data4dappl#78]
in the organization page
https://www.odata.org.il/organization/freedom-of-information-israel
![image](https://user-images.githubusercontent.com/30901226/45254140-d4f55f80-b37b-11e8-8223-4b50ec6bb153.png)

and in the dataset page
https://www.odata.org.il/dataset/post-office
![image](https://user-images.githubusercontent.com/30901226/45254138-c9099d80-b37b-11e8-8b9f-041f015d1ff1.png)

should be a text with the name of the  organization while hovering over the logo

in the footer, the logos of facebook and RSS should be bigger and with overlay text so blind people can understand it 

![image](https://user-images.githubusercontent.com/30901226/45263524-30365900-b433-11e8-8d3c-69600ff3d3d0.png)

**Comment by OriHoch** [comment]
done for entity (group) page as well, e.g. - https://www.odata.org.il/he/group/foreign-commonwealth-office

**Comment by OriHoch** [comment]
done + deployed
